src/main/plots/consts.py

```diff
@@ -23,3 +26,47 @@
 DIFF_MAX = 30
 
 
+class STATISTICS_COLUMN(Enum):
+    AGE = 'age'
+    EXPERIENCE = 'experience'
+    FREQ = 'freq'
+
+
+class STATISTICS_SHOWING_KEY(Enum):
+    INCORRECT = 'Incorrect value'
+    NOT_INDICATED = 'Not indicated'
+    OTHERS = 'Others',
+    FREQ = 'Frequency'
+
+
+STATISTICS_RARE_VALUE_BORDER = 2
+
+
+class STATISTICS_COLORS(Enum):
+    PIE_CHART = px.colors.sequential.Sunset
+    BAR_CHART_BG = '#FFFAE3'
+    BAR_CHART_COLS = '#9B0B9B'
+
+
+class PLOT_TYPES(Enum):
+    PIE = 'pie'
+    BAR = 'bar'
+
+
+class STATISTICS_INFO_FOR_PLOTS(Enum):
+    LABELS = 'labels'
+    TITLE = 'title'
+
+
+class OUTPUT_FORMAT(Enum):
+    HTML = 'html'
```

Comment:
Even if it works without the dot, let's add a dot to the extension to match file_util format?